spec/factories/static_slot_members.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :static_slot_member do
+  end
```

Comment:
Is this empty factory valid? I thought that it belonged to `member` and `static_slot`?